frontend/tests/controllers/ProblemUpdateTest.php

```diff
@@ -2506,4 +2506,107 @@ public function testUpdateStatementProblemViaSubmit() {
             $response['statement']['markdown']
         );
     }
+
+    public function testCommitModifiedZipExcludesEasyCases(): void {
+        // Get a problem
+        $problemData = \OmegaUp\Test\Factories\Problem::createProblem();
+        $problem = $problemData['problem'];
+        $identity = $problemData['author'];
+
+        //Get the current ZIP file with all the files
+        $artifacts = new \OmegaUp\ProblemArtifacts(
+            $problem->alias
+        );
```

Comment:
Esto cabe prefecto en una línea:
```suggestion
        $artifacts = new \OmegaUp\ProblemArtifacts($problem->alias);
```